corehq/apps/cloudcare/urls.py

```diff
@@ -8,13 +8,7 @@
     url(r'^(?P<urlPath>.*)$', 'cloudcare_main', name='cloudcare_main'),
 )
 
-cases_urls = patterns('corehq.apps.cloudcare.views',
-    url(r'^view/(?P<case_id>[\w-]*)/$', 'view_case', name='cloudcare_view_case'),
-    url(r'^create/$', 'view_case', {'case_id': None}, name='cloudcare_create_case'),
-)
-
 api_urls = patterns('corehq.apps.cloudcare.views',
-    url(r'^groups/(?P<user_id>[\w-]*)/$', 'get_groups', name='cloudcare_get_groups'),
```

Comment:
I sanity checked that these weren't being used on the proxy just to be sure:

czue@hqproxy0:~$ grep /cloudcare/cases/  /home/cchq/www/production/log/cchq-apache.log | wc
      0       0       0
czue@hqproxy0:~$ grep /cloudcare/api/groups/  /home/cchq/www/production/log/cchq-apache.log | wc
      0       0       0

(and to verify the command works)
czue@hqproxy0:~$ grep /cloudcare/apps/  /home/cchq/www/production/log/cchq-apache.log | wc
 457248 9614334 150826846